Bounding_volumes/include/CGAL/Min_annulus_d.h

```diff
@@ -21,6 +21,10 @@
 #ifndef CGAL_MIN_ANNULUS_D_H
 #define CGAL_MIN_ANNULUS_D_H
 
+#ifdef _MSC_VER
+# pragma warning(disable: 4244) // conversion warning in Boost iterator_adaptor
+#endif
+
```

Comment:
We cannot accept that! Warning should be disable only locally.

It can be done with:
```C++
#ifdef _MSC_VER
#  pragma warning(push)
#  pragma warning(disable: 4244)
#endif

#ifdef _MSC_VER
#  pragma warning(pop)
#endif
```

Otherwise the warning will be removed for anything that is included after.